packages/browserify/src/index.js

```diff
@@ -111,19 +111,19 @@ function getConfigurationFromPluginOpts (pluginOpts) {
     pr: 'includePrelude',
     prelude: 'includePrelude',
     pp: 'prunePolicy',
-    prunepolicy: 'prunePolicy',
```

Comment:
Are the browserify changes intimately linked to the viz changes? If not can they be broken out to a separate PR?